src/core/connection.c

```diff
@@ -117,6 +117,7 @@ QuicConnAlloc(
     QuicSendInitialize(&Connection->Send);
     QuicLossDetectionInitialize(&Connection->LossDetection);
     QuicDatagramInitialize(&Connection->Datagram);
+    QuicRundownAcquire(&Session->Registration->ConnectionRundown);
```

Comment:
Server side connections cannot acquire the ref in alloc, as they don't have a registration associated until they've been accepted by a listener.